File: app/src/androidTest/java/com/androsaces/eyec/MainActivityTest.java
package com.androsaces.eyec;

import androidx.test.ext.junit.rules.ActivityScenarioRule;
import androidx.test.ext.junit.runners.AndroidJUnit4;
import org.junit.Rule;
import org.junit.Test;
import org.junit.runner.RunWith;

import static androidx.test.espresso.Espresso.onView;
import static androidx.test.espresso.assertion.ViewAssertions.matches;
import static androidx.test.espresso.matcher.ViewMatchers.withId;
import static androidx.test.espresso.matcher.ViewMatchers.withText;

@RunWith(AndroidJUnit4.class)
public class MainActivityTest {
    @Rule
    public ActivityScenarioRule<MainActivity> mActivityRule = new ActivityScenarioRule<>(MainActivity.class);

    @Test
    public void buttonDisplaysCorrectText() {
        onView(withId(R.id.countdownButton)).check(matches(withText("14 days remaining")));
    }
}

File: app/src/androidTest/java/com/androsaces/eyec/MainActivityTest.kt
package com.androsaces.eyec

import androidx.test.espresso.Espresso
import androidx.test.espresso.ViewInteraction
import androidx.test.espresso.assertion.ViewAssertions
import androidx.test.espresso.matcher.ViewMatchers
import org.junit.Test

class MainActivityTest {
    @Test
    fun buttonDisplaysCorrectText() {
        val button: ViewInteraction = Espresso.onView(ViewMatchers.withId(R.id.countdownButton))
        button.check(ViewAssertions.matches(ViewMatchers.withText("14 days remaining")))
    }
}